### PR TITLE
updates to Using content for Zowe CLI V3

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Updating Zowe CLI content for Zowe V3 in Using tab

List the file(s) included in this PR: file.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Review by adam-wolfe — APPROVED** [review]
LGTM

File: docs/user-guide/cli-using-understand-profiles-configs.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ The table below shows how Zowe CLI determines which profiles, properties, and va
...
 
-<b>*</b> If the same profile exists in both a global configuration file and a project configuration file, the project configuration profile completely replaces the global profile. This is true even when the project profile has fewer properties in the same profile found in the global file.<br/><br/>
+<b>*</b> If the same profile exists in both a global configuration file and a project configuration file, the project configuration profile completely replaces the global profile. This is true even when the project profile has fewer properties in the same profile found in the global file.<br/>
 
 The rules above apply when profiles have the same name. To maintain the same set of properties in two different profiles, give each profile a different name so that Zowe CLI uses a specific profile, if needed.
```
Comment (sbohnert) [inline docs/user-guide/cli-using-understand-profiles-configs.md:111]: Replace "above" with "the previous rules" or move this line prior to the rules and say something like "the following rules."

File: docs/user-guide/cli-using-writing-scripts.md [hunk 2]
```diff
@@ -2,19 +2,35 @@
...
 You can combine multiple Zowe CLI commands in bash or shell scripts to automate actions on z/OS. Implement scripts to enhance your development workflow, automate repetitive test or build tasks, and orchestrate mainframe actions from continuous integration/continuous deployment (CI/CD) tools such as Jenkins or TravisCI.
 
-**Note:** The type of script that you write depends on the programming languages that you use and the environment where the script is executed. The following is a general guide to Zowe CLI scripts. Refer to third-party documentation to learn more about scripting in general.
+:::note
+
+The type of script that you write depends on the programming languages that you use and the environment where the script is executed. The following is a general guide to Zowe CLI scripts. Refer to third-party documentation to learn more about scripting in general.
+
+:::
 
 **Follow these steps:**
```
Comment (sbohnert) [inline docs/user-guide/cli-using-writing-scripts.md:11]: Replace "Follow these steps:" with "To write a script:"

**Review by sbohnert — APPROVED** [review]
Looks good. Thanks

**Pull request merged**